packages/matter-node.js/test/IntegrationTest.ts

```diff
@@ -149,30 +149,30 @@ describe("Integration", () => {
             assert.equal(nodeId.id, BigInt(1));
         }, 60 * 1000 /* 1mn timeout */);
 
-        it("the session is resumed if it has been established previously", () => {
-            client.connect(new NodeId(BigInt(1)));
+        it("the session is resumed if it has been established previously", async () => {
+            await client.connect(new NodeId(BigInt(1)));
```

Comment:
Perhaps replace `BigInt(1)` with `1n`?

Reply:
From my knowledge full Bigint support came in ES2020 ... but our typescript setting generates ES2015 ... with "BigInt()" there are no issues ... just the "1n" notation

Reply:
(and we randomize the ID with one of the next PRs anyway" :-)